### DISCUSSION TITLE
Use data from cloud

### DISCUSSION THREAD

**Discussion by giusmar** [nextflow-io/nextflow#2606] [Q&A] [answered]
Hi, i'm exploring all the features of nextflow and i want to use data stored on gscloud. So, when i create a paramsfile.yml, instead of set local directory for file, as fastq.gz files, i wanna put the full path of gscloud. 
All i can found are tutorial how to run nextfow on Google cloud platform, but i can't find an example for my situation. Can someone help me, please? Explaing me all steps.

Thank you for the great work of nextflow :)

**Comment by pditommaso** [comment, marked as answer]
To use data on Google storage all you need to do is provide a `gs://` prefixed file path (plus providing the corresponding creds).

More details look [here](https://www.nextflow.io/docs/latest/google.html#credentials) (if you look at that docs please ignore the section where it says to export the variables NXF_VER=20.01.0
NXF_MODE=google, it must be updated)

**Reply by giusmar** [reply]
Ah ok ty.
I was distracted for the initial part, where ive to ignore the part of nextflow version.
So, after created the json file from google cloud, i can use the file directly from gs also putting the path in a file.yml as a normal params for nextflow. Ty so much.
I'll update this reply after i've obtained the credential for gcloud and completed a run with file from gcloud. :)

**Reply by giusmar** [reply]
Problem solved. It is more complex create a good JSON file with permission from Google cloud than reorganize all works on nextflow.

I want to underline that the section in wich are specified to use an older versione of nextflow confused me.

Bye :)
